docs/eventing/samples/helloworld/helloworld-python/README.md

```diff
@@ -204,6 +211,15 @@ We can send an http request directly to the Broker with correct CloudEvent heade
 
       exit
       ```
+
+**Note** : For v0.18 or above, use `http://broker-ingress.knative-eventing.svc.cluster.local/knative-samples/default` as the address insteed of `default-broker.knative-samples.svc.cluster.local` . Your can view the addree by :
```

Comment:
insteed -> instead
remove the space after the default-broker... and '.'.
Your can -> You can
addree -> address
and also remove the space between 'by' and ':'.

Reply:
It's my fault, fixed writing.